setup.py

```diff
@@ -43,7 +44,8 @@ def visit_Assign(self, node):
 def read_version():
     """Read version from httpretty/version.py without loading any files"""
     finder = VersionFinder()
-    finder.visit(ast.parse(local_file('httpretty', '__init__.py')))
+    finder.visit(
+        ast.parse(local_file('httpretty', '__init__.py').encode('utf-8')))
```

Comment:
We have to encode this away from a unicode string, because otherwise we hit "SyntaxError: encoding declaration in Unicode string".